qcelemental/tests/test_constants.py

```diff
@@ -96,3 +96,10 @@ def test_codata2022():
         qcelemental.PhysicalConstantsContext("CODATA2022")
 
     assert "only contexts {'CODATA2014', 'CODATA2018', } are currently supported" in str(e.value)
+
+
+def test_codata_comparison():
+    old = qcelemental.constants
+    new = qcelemental.PhysicalConstantsContext("CODATA2018")
```

Comment:
We should be explicit where we check `old= qcelemental.PhysicalConstantsContext("CODATA2014")` incase our defaults change.

Reply:
only in the tests for coverage, but yep.